{{cookiecutter.project_slug}}/config/settings/production.py

```diff
@@ -182,12 +182,14 @@
         'PASSWORD': env('RDS_PASSWORD'),
         'HOST': env('RDS_HOSTNAME'),
         'PORT': env('RDS_PORT'),
+        'CONN_MAX_AGE': env.int('CONN_MAX_AGE', default=60),
     }
 }
 {% else %}
 # Use the Heroku-style specification
 # Raises ImproperlyConfigured exception if DATABASE_URL not in os.environ
 DATABASES['default'] = env.db('DATABASE_URL')
+DATABASES['default']['CONN_MAX_AGE'] = env.int('CONN_MAX_AGE', default=60)
```

Comment:
The `default` argument should be refactored to avoid duplication, IMO.

Reply:
Make it smth like `_DEFAULT_CONN_MAX_AGE` so the name indicates its intricate nature.

Reply:
Is there a way to set this globally without adding to the list of questions? I know of Jinja's `{% set ... %}` but that would still be duplicated between the 2 files `production.py` and `.env`.

Reply:
@browniebroke I don't think we need to make this a cookiecutter option -- merely introducing `_DEFAULT_CONN_MAX_AGE = 60` to `production.py` should suffice; as for the `env.example`, I suggest that you leave its `CONN_MAX_AGE` blank (`CONN_MAX_AGE=`).

Reply:
Ok agreed, I will do that. Thanks for your help.